### ISSUE TITLE
[Android] Add LanChat to Share menu

### ISSUE THREAD

**Issue by ialokim** [sebkur/lanchat#3]
It would be nice to directly share from other apps to the local network via LanChat. Until #2 is supported, this could only be offered when sharing text (probably using some MIME/type filter).

**Comment by sebkur** [comment]
Good idea. I think we need to do what's outlined [here](https://developer.android.com/training/sharing/receive) for this.
